src/plot.js

```diff
@@ -354,6 +351,20 @@ export function plot(options = {}) {
   return figure;
 }
 
+function createTitleElement(contents, tag, className) {
+  if (contents.ownerDocument) return contents;
+  const e = document.createElement(tag);
```

Comment:
This needs to use context.document instead of the global document.

Reply:
oops! good catch, thanks!